governance/polls/MIP9 Community Greenlight Poll - aUST (aUST) - December 06, 2021.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Community Greenlight Poll - aUST (aUST) - December 06, 2021
+title: MIP9 Community Greenlight Poll - aUST (aUST) - December 06, 2021
```

Comment:
Only needed MIP9 in the file name, not the title section. Should have been clearer, sorry

Reply:
I'm also going to be really annoying and ask can we change all "aUST (aUST)" to "aUST (Anchor TerraUSD)" as that's probably a more descriptive asset name. I should have been clearer about this earlier tbf